src/Api/index.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const apis = {
+	verifyUser(token) {
+		return axios
+			.get(`http://eagle-bn-backend-staging.herokuapp.com/api/v1/users/verify/${token}`)
+			.then(result => {
+				return result.data;
+			})
+			.catch(err => {
+				return err.response;
+			});
```

Comment:
I highly recommend that we export these axios function as `async` like:
```
async verifyUser(token) => axios.get(link + token)
```
then we can catch errors within the `actions` themselves